Xamarin.Forms.Xaml.UnitTests/SetValue.xaml.cs

```diff
@@ -243,6 +275,38 @@ public void MorePrimitiveTypes(bool useCompiledXaml)
 				Assert.AreEqual((ushort)32, page.mockView0.UShort);
 				Assert.AreEqual((decimal)42, page.mockView0.ADecimal);
 			}
+
+			[TestCase(false)]
+			[TestCase(true)]
+			public void SetValueWithImplicitOperatorOnSource(bool useCompiledXaml)
+			{
+				var page = new SetValue(useCompiledXaml);
+				Assert.AreEqual("Bar", page.implicit0.GetValue(MockViewWithValues.BPBarProperty));
+			}
+
+			//[TestCase(false)]
+			//[TestCase(true)]
+			//public void SetValueWithImplicitOperatorOnTarget(bool useCompiledXaml)
+			//{
+			//	var page = new SetValue(useCompiledXaml);
+			//	Assert.AreEqual("Foo", ((SV_Foo)page.implicit1.GetValue(MockViewWithValues.BPFooProperty)).Value);
+			//}
+
+			[TestCase(false)]
+			[TestCase(true)]
+			public void SetWithImplicitOperatorOnSource(bool useCompiledXaml)
+			{
+				var page = new SetValue(useCompiledXaml);
+				Assert.AreEqual("Bar", page.implicit2.Bar);
+			}
+
+			//[TestCase(false)]
+			//[TestCase(true)]
+			//public void SetWithImplicitOperatorOnTarget(bool useCompiledXaml)
+			//{
+			//	var page = new SetValue(useCompiledXaml);
+			//	Assert.AreEqual("Foo", page.implicit3.Foo.Value);
+			//}
```

Comment:
not supported on Xaml-nonC right now

it's a placeholder for when I'll come back to this to fix it, not a leftover